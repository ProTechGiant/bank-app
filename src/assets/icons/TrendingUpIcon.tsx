import React from "react";
import Svg, { Path, Rect } from "react-native-svg";

import { IconProps } from ".";

export function TrendingUpIcon({ color = "#00A0CC", height = 24, width = 24 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Rect width="44" height="44" rx="22" fill="#F5F5F5" />
      <Path
        d="M25.3337 17L27.242 18.9083L23.1753 22.975L19.842 19.6417L13.667 25.825L14.842 27L19.842 22L23.1753 25.3333L28.4253 20.0917L30.3337 22V17H25.3337Z"
        fill={color}
      />
    </Svg>
  );
}
