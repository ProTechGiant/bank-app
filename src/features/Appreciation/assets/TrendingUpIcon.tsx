import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export const TrendingUpIcon = ({ width = 16, height = 16, color = "#FF371E" }: IconProps) => {
  return (
    <Svg width={width} height={height} viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M10.6663 4L12.193 5.52667L8.93967 8.78L6.27301 6.11333L1.33301 11.06L2.27301 12L6.27301 8L8.93967 10.6667L13.1397 6.47333L14.6663 8V4H10.6663Z"
        fill={color}
      />
    </Svg>
  );
};
