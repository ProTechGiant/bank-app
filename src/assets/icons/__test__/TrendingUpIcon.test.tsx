import { create } from "react-test-renderer";

import { TrendingUpIcon } from "../TrendingUpIcon";

describe("icon/TrendingUpIcon", () => {
  it("renders a TrendingUpIcon", () => {
    const result = create(<TrendingUpIcon />);

    expect(result).toMatchSnapshot();
  });
});
