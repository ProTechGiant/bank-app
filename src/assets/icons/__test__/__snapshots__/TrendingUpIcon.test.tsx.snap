// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/TrendingUpIcon renders a TrendingUpIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={44}
  vbWidth={44}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGRect
      fill={
        {
          "payload": 4294309365,
          "type": 0,
        }
      }
      height="44"
      propList={
        [
          "fill",
        ]
      }
      rx="22"
      width="44"
      x="0"
      y="0"
    />
    <RNSVGPath
      d="M25.3337 17L27.242 18.9083L23.1753 22.975L19.842 19.6417L13.667 25.825L14.842 27L19.842 22L23.1753 25.3333L28.4253 20.0917L30.3337 22V17H25.3337Z"
      fill={
        {
          "payload": 4278231244,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
